Tasks/DownloadPackageV1/make.json

```diff
@@ -0,0 +1,11 @@
+{
+    "rm": [
+        {
+            "items": [
+                "node_modules/azure-pipelines-tasks-packaging-common/node_modules/azure-pipelines-task-lib",
+                "node_modules/azure-pipelines-tool-lib/node_modules/azure-pipelines-task-lib",
+                "node_modules/azure-pipelines-tasks-utility-common/node_modules/azure-pipelines-task-lib"
+            ],
+            "options": "-Rf"
+        }    ]
```

Comment:
Please fix the json formatting, closing array should be on next line, indented appropriately

Reply:
Fixed, thanks!